tools/mo/openvino/tools/mo/middle/layer_normalization.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+from openvino.tools.mo.front.caffe.extractors.utils import get_canonical_axis_index
+from openvino.tools.mo.front.common.partial_infer.utils import int64_array
+from openvino.tools.mo.front.tf.graph_utils import create_op_node_with_second_input
+from openvino.tools.mo.graph.graph import Graph, rename_nodes
+from openvino.tools.mo.middle.replacement import MiddleReplacementPattern
+from openvino.tools.mo.ops.elementwise import Mul, Add
+from openvino.tools.mo.ops.mvn import MVN
+from openvino.tools.mo.ops.unsqueeze import Unsqueeze
+
+
+class LayerNormalization(MiddleReplacementPattern):
+    """
+    Decompose LayerNorm(x) to MVN(x) * gamma + beta
```

Comment:
please add a restriction here that currently we support LayerNorm with only 0 output port. Update documentation with this restriction as well.

Reply:
BTW, mean can be computed with ReduceMean